plasmapy/__init__.py

```diff
@@ -51,12 +51,9 @@
 
 __name__ = "plasmapy"
 
-class UnsupportedPythonError(Exception):
-    pass
-
 
 if sys.version_info < tuple((int(val) for val in "3.6".split('.'))):
-    raise UnsupportedPythonError("plasmapy does not support Python < {}".format(3.6))
+    raise RuntimeError("plasmapy does not support Python < {}".format(3.6))
```

Comment:
I can't help but think that this line would be better with f-strings...even though it's an error message specifically for the times of eons past before f-strings came into being.  Alas!

Reply:
Not much we can do here :(